src/uenv/log.h

```diff
@@ -0,0 +1,6 @@
+#include <spdlog/spdlog.h>
+
+namespace uenv {
+void init_log(spdlog::level::level_enum console_log_level,
+              spdlog::level::level_enum syslog_log_level);
```

Comment:
I don't love this signature because the levels can be passed in the wrong order without anyone noticing.

I'm considering adding some strong types just for this interface, but it does also add a bit of complexity. @bcumming do you think it's worth making this safer or shall we keep it simple for now? We likely won't have many calls to `init_log`...

Reply:
Keep it simple for now.